app/actions/service_credential_binding_create.rb

```diff
@@ -84,7 +84,7 @@ def all_space_guids(service_instance)
       end
 
       def event_repository
-        Repositories::ServiceBindingEventRepository
+        Repositories::ServiceGenericBindingEventRepository.new('service_binding')
```

Comment:
Should the parameter be a constant defined somewhere in the `ServiceGenericBindingEventRepository`? I can see a world where we decide to update it to something different and have to remember to change in multiple places 🤔